### ISSUE TITLE
V1?

### ISSUE THREAD

**Issue by espenstoro** [hammondeggs/hammondeggsModules#2]
Hi. Do you have any plans to migrate to Rack 1.0? I really miss the BBD modules, nothing else does what they do.

**Comment by hammondeggs** [comment]
I do plan on it - a: was sort of waiting for the teething issues to be sorted out, and b: now that it’s summer I’ll have a bit of time to get on that!

Thanks ;
________________________________

**Comment by espenstoro** [comment]
Great to hear, looking forward to it.

**Comment by dragnett** [comment]
yes, please. I really want to try out BBD for a retro style project, Looking forward to V1 and any new additions , A million cheers,...  Jeff S.

**Comment by Petervos2018** [comment]
And another plea for BBD ;)

**Comment by dragnett** [comment]
vcvrack 1.+ is rocking now, Please please please port the BBD, (plus phase3) Please.. :)

**Comment by atarv** [comment]
Have you considered open sourcing these plugins, if you don't have time/whatever to update to vcvrack 1.*?

**Comment by hammondeggs** [comment]
Porting to 1.0 for BBD and Phase 3 is complete. These will be released soon. Sorry for the delay!

**Comment by persywho** [comment]
have these v1 ports been released yet? any plans to port the rest of the modules? would love to see the whole plugin ported over. and maybe you could think about adding your Chip2.0 from your loguesdk work to the collection.

**Comment by espenstoro** [comment]
Is this one still happening? If not, hand it over to someone who can make a port to v1? It's been a year...

**Comment by Ahornberg** [comment]
Do you plan to open the source code so that other devs can port your plugin to VCV 1.+?

**Comment by 9Skrip** [comment]
dear developer please update or opensourse
